docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,215 @@
+      10. For `Number of replicas`, enter `2`.
+      11. For high availability, select the `Multi AZ` checkbox. Take a note that this may almost double your monthly expense. More details [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html).
+      12. Select `Advanced Redis settings` and fill in the details as mentioned below.
+          1. For `Subnet group`, create a new group.
+          2. Add the subnets that were created above.
+          3. For `VPC ID`, add the VPC that was created above.
+          4. For `Availability zone(s)`, choose `No preference`. More details [here](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/RegionsAndAZs.html).
+          5. For `Security groups` choose the default security group.
+      13. Click the `Create` button.
+      14. Proceed when the cluster's status becomes `available`. Next we'll grant access to the cluster.
+      15. Go to the AWS EC2 console [here](https://console.aws.amazon.com/ec2/).
+      16. In the navigation pane, under `Network & Security`, choose `Security Groups`.
+      17. Choose the `default` security group for your VPC.
+      18. Choose the `Inbound` tab, and then do the following:
+          1. Choose `Edit` and select `Add Rule`.
+          2. In the `Type` column, choose `Custom TCP rule`.
+          3. In the `Port range` box, type `6379`.
+          4. In the `Source` box, choose `Anywhere`.
+          5. Click `Save`.
+      19. Find the endpoints of your redis instance using [this](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Endpoints.html) and copy them somewhere for later use.
+
+11. Connect the redis instance to the `plio-analytics-staging` service that was created above.
+    1. Go to the ECS dashboard and select `plio-staging-cluster`.
+    2. Select the `plio-analytics-staging` service.
+    3. Select the `Tasks` tab.
+    4. Select the running task definition in the `Task Definition` column.
+    5. Click `Create new revision` button on the top and select the existing container with the name: `plio-analytics-staging`.
+    6. In the new window, scroll down to `Environment Variables` section.
+    7. Update the `CUBEJS_REDIS_URL` variable that were copied before. By default, AWS Elasticache has no password. In case you had configured it, make sure to update `CUBEJS_REDIS_PASSWORD` variable as well.
+    8. Click the `Update` button to update the container definition.
+    9. Click the `Create` button to create a new revision of the task definition.
+    10. Navigate back to the service `plio-analytics-staging`.
+    11. Click `Update` on top right.
+    12. Update the revision of the newly created task definition and wait for it to get initiated.
+    13. Wait for some time. The older task will automatically stop once the new task reaches a stable state.
```

Comment:
Does this happen automatically? I remember having to manually kill the older tasks - maybe I didn't wait long enough?

Reply:
yeah it happens automatically. I was also stopping tasks manually but later figured out that it happens automatically. just need to wait for some time.